### PR TITLE
feat: use auto-generated AlloyDB client

### PR DESCRIPTION
**Pull request by jackwotherspoon**
Use auto-generated client for requests: https://pkg.go.dev/cloud.google.com/go/alloydb

### CODE REVIEW

File: internal/alloydb/refresh.go [hunk 1]
```diff
@@ -27,9 +27,11 @@ import (
 	"fmt"
 	"time"
 
+	alloydbadmin "cloud.google.com/go/alloydb/apiv1beta"
+	alloydbpb "cloud.google.com/go/alloydb/apiv1beta/alloydbpb"
```
Comment (enocom) [inline internal/alloydb/refresh.go:31]: Is the alias `alloydbpb` is unnecessary here?

File: internal/alloydb/refresh.go [hunk 2]
```diff
@@ -27,9 +27,11 @@ import (
...
 	"cloud.google.com/go/alloydbconn/errtype"
-	"cloud.google.com/go/alloydbconn/internal/alloydbapi"
 	"cloud.google.com/go/alloydbconn/internal/trace"
+	durationpb "google.golang.org/protobuf/types/known/durationpb"
```
Comment (enocom) [inline internal/alloydb/refresh.go:34]: Same here. Looks like the alias is not necessary, unless I'm missing something.

File: internal/alloydb/refresh.go [hunk 3]
```diff
@@ -97,7 +104,14 @@ func fetchEphemeralCert(
 	if err != nil {
 		return certChain{}, err
 	}
-	resp, err := cl.GenerateClientCert(ctx, inst.project, inst.region, inst.cluster, buf.Bytes())
+	req := &alloydbpb.GenerateClientCertificateRequest{
+		Parent: fmt.Sprintf(
+			"projects/%s/locations/%s/clusters/%s", inst.project, inst.region, inst.cluster,
```
Comment (enocom) [inline internal/alloydb/refresh.go:109]: `inst.String()` should produce the same result.

**Review by enocom — APPROVED** [review]
A few minor nits. Otherwise LGTM. :shipit:

File: internal/alloydb/refresh.go (hunk 3 above)
Comment (jackwotherspoon) [reply internal/alloydb/refresh.go:109]: as discussed offline, `inst.String()` does not return the same result, it excludes the words `projects` , `locations` etc.

**Pull request merged**